static/bidder-info/aidem.yaml

```diff
@@ -0,0 +1,19 @@
+endpoint: "https://zero.aidemsrv.com/ortb/v2.5/bid/request"
+maintainer:
+  email: prebid@aidem.com
```

Comment:
@gsollazzo an email is sent by Prebid team to verify maintainer's email address. Please respond to email with "received" message to complete verification